tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -121,13 +122,10 @@ public:
    Long64_t Merge(TCollection *input, TFileMergeInfo *mergeInfo);
 
    /// NOTE: if you change this version you also need to update:
-   ///    - RTFNTuple::fClassVersion in RMiniFile.cxx
-   ///    - RTFStreamerInfoObject::fVersionRNTuple in RMiniFile.cxx
-   ///    - RTFStreamerInfoObject::fStreamers in RMiniFile.cxx
-   ClassDefNV(RNTuple, 6);
+   ///    - RTFNTuple::fVersionClass in RMiniFile.cxx
+   ClassDefNV(RNTuple, 2);
```

Comment:
Why version 2 in the latest PR? Should be documented and at least updated in the commit message...

Reply:
(this also seems to cause the failure in `ntuple_storage`:
```
TClass::RegisterStreamerInfo' reading 'Register StreamerInfo for ROOT::RNTuple on non-empty slot (2).
```
that I don't understand...)

Reply:
This follows the suggestion of @pcanal and seems to be the general recommendation for the first class version of ROOT classes.

The failure comes from the fact that the regenerated RC3 file on root.cern.ch/files still uses class version 1. Updating the file currently has to go through @dpiparo so I'll wait a little to see if more things accumulate.

Reply:
Thanks for noticing the commit message! Updated.